### ISSUE TITLE
When always_specify_types and avoid_return_types_on_setters are both on, don't ask for types on setters

### ISSUE THREAD

**Issue by Hixie** [dart-lang/linter#202]
This blocks https://github.com/flutter/flutter/pull/2666.

**Comment by zoechi** [comment]
I think this is fixed already #192
See also #122

**Comment by Hixie** [comment]
great!

**Comment by Hixie** [comment]
Any idea what version that will be available in? We're on 1.15.0-dev.5.0 and have this problem.

**Comment by pq** [comment]
This should be integrated now.

```
[~/src/git/clones/flutter] (master) $ ./run.sh
Building flutter tool...
Analyzing 217 entry points...
[lint] Avoid return types on setters. (/Users/pquitslund/src/git/clones/flutter/dev/benchmarks/complex_layout/lib/main.dart, line 33, col 3)
[lint] Avoid return types on setters. (/Users/pquitslund/src/git/clones/flutter/dev/benchmarks/complex_layout/lib/main.dart, line 33, col 3)
...
```
(And no noise from `always_declare_return_types`.)
